src/main/asciidoc/_chapters/developer.adoc

```diff
@@ -503,6 +503,40 @@ If you see `Unable to find resource 'VM_global_library.vm'`, ignore it.
 It's not an error.
 It is link:https://issues.apache.org/jira/browse/MSITE-286[officially ugly] though.
 
+[[build.on.linux.aarch64]]
+=== Build On Linux Aarch64
+Hbase runs on both Windows and UNIX-like systems, and it should run on any platform 
+that runs a supported version of Java. This should include JVMs on x86_64 and aarch64. 
+The documentation below describes how to build hbase on aarch64 platform.
+
+==== Set Environment Variables
+Manually install Java and Maven on aarch64 servers if they are not installed, 
+and set environment variables. For example:
+
+[source,bourne]
+----
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-arm64
+export MAVEN_HOME=/opt/maven
+export PATH=${MAVEN_HOME}/bin:${JAVA_HOME}/bin:${PATH}
+----
+
+==== Use Protobuf Supported On Aarch64
+Now hbase uses protobuf of two versions. Version '3.11.4' of protobuf that hbase uses 
+internally and version '2.5.0' as external usage. 
+Package protobuf-2.5.0 does not work on aarch64 platform, we should add maven 
```

Comment:
On master branch we do not depend on protoc-2.5.0 any more? We only depend on protobuf-java 2.5.0 as the hadoop we depend on still uses protobuf-2.5.0.

Reply:
I think now we use protoc 2.5.0 on master, see:
https://github.com/apache/hbase/blob/bc9184ee001c848518440c7e4dfa6e258a82089a/pom.xml#L927
the profile ‘-Paarch64’ is only to use protoc 2.50 of openlabtesting.
And we use protobuf-java 2.50 on mater too.
Or if I missed something?